doc/api/fs.md

```diff
@@ -545,6 +545,9 @@ the end of the file.
 <!-- YAML
 added: v10.0.0
 changes:
+  - version: v15.14.0
+    pr-url: https://github.com/nodejs/node/pull/37490
+    description: The `data` argument supports `AsyncIterable`, `Iterable` & `Stream`.
```

Comment:
```suggestion
    description: The `data` argument supports `AsyncIterable`, `Iterable` and `Stream`.
```

Reply:
It's copied from https://github.com/nodejs/node/blob/7ca38f05a023666274569343f128c5aed81599f3/doc/api/fs.md#L1342-L1344

Reply:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand. Should it be changed in another PR?

Reply:
I guess I can changed it in both places. Thanks for the suggestion.